src/services/completions.ts

```diff
@@ -1529,6 +1541,26 @@ namespace ts.Completions {
             });
         }
 
+        function getVariableDeclaration(property: Node): VariableDeclaration | undefined {
+
+            const isArrowFunctionExpressionWithoutArrow = (node: Node) => {

```

Comment:
Without arrow? 😄 

```suggestion
            const isArrowFunctionBody = (node: Node) => {
```

Reply:
hhhh, I made a mistake. I wanted to write withoutBraces.